test/core/Leaf.t.sol

```diff
@@ -186,88 +186,76 @@ contract LeafTest is Test2 {
 }
 
 contract TickTest is Test {
```

Comment:
Some of these tests are "bin" tests, so maybe either split or rename this contract?

Reply:
renamed in 59bf144b5b0181035b0ac48b8204918a0aee0cc6